diff --git a/gst-env.py b/gst-env.py
index df3e402..f585736 100755
--- a/gst-env.py
+++ b/gst-env.py
@@ -291,6 +291,12 @@ def get_subprocess_env(options, gst_version):
     prepend_env_var(env, "GST_PLUGIN_PATH", os.path.join(options.builddir, 'subprojects',
                                                          'libnice', 'gst'),
                     options.sysroot)
+    prepend_env_var(env, "GST_PLUGIN_PATH", os.path.join(options.builddir, 'subprojects',
+                                                         'usrsctp', 'gst'),
+                    options.sysroot)
+    prepend_env_var(env, "GST_PLUGIN_PATH", os.path.join(options.builddir, 'subprojects',
+                                                         'usrsctp', 'gst'),
+                    options.sysroot)                
     prepend_env_var(env, "GST_VALIDATE_SCENARIOS_PATH",
                     os.path.join(PREFIX_DIR, 'share', 'gstreamer-1.0',
                                  'validate', 'scenarios'),
diff --git a/meson.build b/meson.build
index 303dba1..92024ac 100644
--- a/meson.build
+++ b/meson.build
@@ -74,6 +74,7 @@ subprojects = [
   ['gst-plugins-base', {'option': get_option('base'), 'build-hotdoc': true}],
   ['gst-plugins-good', {'option': get_option('good'), 'build-hotdoc': true}],
   ['libnice', { 'option': get_option('libnice'), 'match_gst_version': false}],
+  ['usrsctp', { 'option': get_option('usrsctp'), 'match_gst_version': false}],
   ['gst-plugins-bad', { 'option': get_option('bad'), 'build-hotdoc': true}],
   ['gst-plugins-ugly', { 'option': get_option('ugly'), 'build-hotdoc': true}],
   ['gst-libav', { 'option': get_option('libav'), 'build-hotdoc': true}],
diff --git a/meson_options.txt b/meson_options.txt
index 818855a..ce90256 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -2,6 +2,7 @@
 option('python', type : 'feature', value : 'auto')
 option('libav', type : 'feature', value : 'auto')
 option('libnice', type : 'feature', value : 'auto')
+option('usrsctp', type : 'feature', value : 'auto')
 option('base', type : 'feature', value : 'enabled')
 option('good', type : 'feature', value : 'enabled')
 option('ugly', type : 'feature', value : 'auto')
